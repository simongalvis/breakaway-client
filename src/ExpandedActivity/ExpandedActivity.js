import React from 'react';
import './ExpandedActivity.css';

class ExpandedActivity extends React.Component{


    render(){
        return(
            <h2>Expanded Activity</h2>
        )
    }
}

export default ExpandedActivity;