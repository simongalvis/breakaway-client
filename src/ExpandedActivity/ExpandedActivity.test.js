import React from 'react'
import ReactDOM from 'react-dom'
import ExpandedActivity from './ExpandedActivity'
import renderer from 'react-test-renderer';
import {BrowserRouter} from 'react-router-dom'

describe(`ExpandedActivity component`, () =>{
  it(`renders without crashing`, () =>{
    const div = document.createElement('div');
  
    ReactDOM.render(<BrowserRouter><ExpandedActivity/></BrowserRouter>, div);
  
    ReactDOM.unmountComponentAtNode(div);
  })
  it(`Renders UI as expected`, () =>{
    const tree = renderer
      .create(<BrowserRouter><ExpandedActivity/></BrowserRouter>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  })
})
