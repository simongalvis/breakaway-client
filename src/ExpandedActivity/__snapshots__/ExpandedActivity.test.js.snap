// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExpandedActivity component Renders UI as expected 1`] = `
<div
  className="ExpandedActivity"
>
  <header>
    <a
      href="/"
      onClick={[Function]}
    >
      <img
        alt="Blue breakaway logo"
        id="main-logo"
        src="breakaway-logo.png"
      />
    </a>
    <a
      href="/login-page"
      onClick={[Function]}
    >
      <button
        className="nav-button"
        id="login-button"
        value="login"
      >
        Log In
      </button>
    </a>
  </header>
  <h2>
    Expanded Activity
  </h2>
</div>
`;
